services/provider/api/client/client.go

```diff
@@ -227,3 +227,19 @@ func (cc *OCSProviderClient) PeerStorageCluster(ctx context.Context, onboardingT
 
 	return cc.Client.PeerStorageCluster(apiCtx, req)
 }
+
+func (cc *OCSProviderClient) RequestMaintenanceMode(ctx context.Context, consumerUUID string, requestFailover bool) (*pb.RequestMaintenanceModeResponse, error) {
```

Comment:
As agreed we should not use 'failover' terminology 
```suggestion
func (cc *OCSProviderClient) RequestMaintenanceMode(ctx context.Context, consumerUUID string, enable bool) (*pb.RequestMaintenanceModeResponse, error) {
```